src/lib.rs

```diff
@@ -89,7 +89,7 @@ impl Target {
     }
 
     /// Size of a pointer in bytes
```

Comment:
@seanyoung This should be "Size of a pointer in bits", shouldn't it?

Reply:
Good catch, yet